sponser_dashboard.html

```diff
@@ -203,97 +18,92 @@
     </head>
     <body>
         <nav>
-            <div style="position: absolute;right: 350px; top: 12px;">
-                <div class="circleoutline">
-                    <button onclick="showLogout()" class="circlebutton" type="button">J</button>
-                    <div id="logout" onclick="window.location.href = 'login.html';">
-                        <i style="color: #C4183C;float: left;padding: 13px 0px 0px 0px;margin-left: 10px;" class="fas fa-sign-out-alt"></i>
-                        <span id="sign-out">Sign Out</span><br>
-                        <p style="font-size: 14px; margin-left: 13px;">john.doe@brainxtech.com</p>
-                    </div>
+            <div class="circleoutline">
+                <button onclick="showLogout()" class="circlebutton" type="button">J</button>
+                <div id="logout" onclick="window.location.href = 'login.html';">
+                    <i class="fas fa-sign-out-alt icon-style"></i>
+                    <span id="sign-out">Sign Out</span><br>
+                    <p class="email-style">john.doe@brainxtech.com</p>
                 </div>
             </div>
+
         </nav>
-        <div style="margin-top: 45px;">
-            <h1 id="fair-heading">Virtual Fair</h1>
-        </div>
-        <div>
-            <div class="column1" style="margin-left: 250px;">
-                <ul>
+        <div class="container">
+            <div >
+                <h1 id="fair-heading">Virtual Fair</h1>
+            </div>
+            
+            <div class="column1">
+                <ul class="fair-tabs">
                     <li><a href="#">Event Detail</a></li>
                     <li><a href="#">Vendor</a></li>
-                    <li><a href="#" style="border-bottom: 3px solid blue;color: blue">Sponser</a></li>
+                    <li><a href="#" class="sponsor-style">Sponser</a></li>
```

Comment:
separate class is used to show active tab.
`active` adding this class should make active and change styling